src/mappers/cms-model.mapper.ts

```diff
@@ -0,0 +1,30 @@
+import { CmsDTO } from '../dtos/cms.dto';
+import { CmsModel } from '../models/cms.model';
+
+export const toModel = (dto: CmsDTO): CmsModel => ({
+  aboutUs: dto.about_us,
+  pages: dto.pages.map((page) => {
```

Comment:
Could you try this:
`
dto.pages.map((page) => ({ label: page.label, ... });
`